### DISCUSSION TITLE
Common Configuration Error

### DISCUSSION THREAD

**Discussion by funilrys** [funilrys/PyFunceble#277] [Q&A]
Hello everyone, 

I'm willing to add some messages to inform user when their setup may produce unsuspected behaviors.

It all started with #274 and it's fixing commit https://github.com/funilrys/PyFunceble/commit/ee875a56adba63186a81e30886f974c562d4b73e, but I need more examples!

Therefore, I invite you to let me know if you have made some mistakes that may help others in the future. 
All feedbacks are welcome!

Stay safe and healthy!
Nissar

**Comment by spirillen** [comment]
An Example could be if you have a fixed or default output directory set than default behavior to Pyfunceble

```
cat ~/.config/PyFunceble/.pyfunceble-env

PYFUNCEBLE_OUTPUT_LOCATION=/tmp/pyfunceble
```

This is a ramfs dir: https://pyfunceble.readthedocs.io/en/latest/facts/faq.html#setup-and-use-ramfs

Here it could be nice if it outputted a message reminding the user that the results would NOT be in the `./output/` but in `/tmp/pyfunceble/output/`

**Reply by spirillen** [reply]
Got another though here.

Generate a output message for every command argument applied that differ from `~/.config/PyFunceble/.PyFunceble.overwrite.yaml`  I'm using the `$USER` config here, as of if this is picked up from a project folder there can be other "switches" than the one a user have as default. While a project can be shared, I think it can be practical and nice to be reminded what the diff is from own default.
